src/main/java/seedu/toluist/ui/UiStore.java

```diff
@@ -24,6 +26,8 @@
  * Ui will observe for changes to `shownTasks` and re-render on state change
  */
 public class UiStore {
+    public static final int INVALID_SUGGESTION_INDEX = -1;
```

Comment:
![screen shot 2017-03-25 at 09 12 59](https://cloud.githubusercontent.com/assets/7934515/24318132/6d59e938-113b-11e7-8c5f-74ca3253a20f.png)
not sure actually